packages/components/src/date-time/test/date.js

```diff
@@ -1,97 +1,118 @@
 /**
  * External dependencies
  */
-import { shallow } from 'enzyme';
 import moment from 'moment';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import 'react-dates/initialize';
 
 /**
  * Internal dependencies
  */
 import DatePicker from '../date';
 
-const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
-
 describe( 'DatePicker', () => {
-	it( 'should pass down a moment object for currentDate', () => {
-		const currentDate = '1986-10-18T23:00:00';
-		const wrapper = shallow( <DatePicker currentDate={ currentDate } /> );
-		const date = wrapper.children().props().date;
-		expect( moment.isMoment( date ) ).toBe( true );
-		expect( date.isSame( moment( currentDate ) ) ).toBe( true );
+	it( 'should highlight the current date', () => {
+		render( <DatePicker currentDate="2022-05-02T11:00:00" /> );
+
+		expect(
+			screen.getByLabelText( 'Monday, May 2, 2022' ).classList
```

Comment:
Nit: `getByRole` could be slightly better than `getByLabelText` because it makes it more readable for which aria role the element has.

Reply:
Oops, missed that these had a role! 👍